icssc-projects/AntAlmanac
# Css fixes

**Summary**
Just some simple css fixes:
- Left and right toolbars are now the same fixed height
- Settings pop up is widened (to a fixed width) to fit the content fully
  - Note: It's a little wide depending on the resolution. I'm not sure if the way I did it is the best
- Added spacing between buttons in: 
  - Calendar pane toolbar
  - SectionTable course info
  - Copy and Clear Schedule in Added Classes tab

![image](https://user-images.githubusercontent.com/25188285/105614467-7c716200-5d7e-11eb-8d2b-f09cd6740878.png)
![image](https://user-images.githubusercontent.com/25188285/105614422-4502b580-5d7e-11eb-85e7-e7acf43f5fb9.png)
![image](https://user-images.githubusercontent.com/25188285/105614446-64014780-5d7e-11eb-8be5-7b74f98511d3.png)
![image](https://user-images.githubusercontent.com/25188285/105614435-564bc200-5d7e-11eb-8f87-6e7e4519b930.png)



**Test Plan**
- Check that the button spacing in each area is adequate
- Try different zooms to see how it'd behave on different screen sizes (for the toolbar height and the settings pop up width)

**Issues**
#36
